### DISCUSSION TITLE
"_protected/app/configs/config.ini" config file cannot be found at first install

### DISCUSSION THREAD

**Discussion by El3ctra** [pH7Software/pH7-Social-Dating-CMS#1154] [Q&A]
Hello
Once the CMS files copied to host directory, I get this error saying the config.ini was not found.
However, after downloading several versions, I'm not finding any config.ini file in the protected foder?
Am I doing something worng?
After some research, constants.php is setting config.ini as a variable
I've found another config.ini in "_install\data\configs". But it doesnt seem to be the right one.
Can you please help me out?
IS there a howto I could check?
Thank you for your help

**Comment by El3ctra** [comment]
I tried a fresh installation with v17 package and it worked fine.

**Reply by lidysun** [reply]
which version ? could you please replay a version link for me ? i meet the same bug and dont know how to fix it . thanks for you help
